### PR TITLE
C365584 Verify  "Save & close" button on the "Location look-up" (fire…

### PR DESCRIPTION
**Pull request by KarapetyanSofia**
[C365584](https://foliotest.testrail.io/index.php?/cases/view/365584)
[Allure Report ](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5289/allure)
image

### CODE REVIEW

File: cypress/support/fragments/settings/tenant/location-setup/locations.js [hunk 1]
```diff
@@ -84,8 +84,8 @@ export default {
     LocationDetails.waitLoading();
     LocationDetails.checkActionButtonAbsent();
   },
-  selectInstitution() {
-    cy.do(Select('Institution').choose(including('KU')));
+  selectInstitution(content) {
+    cy.do(Select('Institution').choose(including(content)));
```
Comment (nayimovag) [inline cypress/support/fragments/settings/tenant/location-setup/locations.js:88]: please if you change methods, make sure it doesn't break other tests

File: cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js [hunk 2]
```diff
@@ -884,7 +885,9 @@ export default {
   isDragAndDropAreaDisabled(isDisabled) {
     cy.expect(fileButton.has({ disabled: isDisabled }));
   },
-
+  isCloseAndSaveButtonDisabled(isDisabled) {
```
Comment (nayimovag) [inline cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js:888]: is it closeAndSave button or saveAndClose button? make sure method names are relevant

File: cypress/support/fragments/settings/tenant/location-setup/locations.js (hunk 1 above)
Comment (KarapetyanSofia) [reply cypress/support/fragments/settings/tenant/location-setup/locations.js:91]: Yes, I`ve already changed the method. thanks!

File: cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js (hunk 2 above)
Comment (KarapetyanSofia) [reply cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js]: Done!

**Review by nayimovag — APPROVED** [review]

**Review by Scorpibear — APPROVED** [review]

**Pull request merged**